src/components/SignUp.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Typography, Button, CssBaseline, TextField, Link, Grid, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
```

Comment:
Here you imported `Container` explicitly. We need to do all our imports like this. Fortunately, VSCode will do it for us. :-)

Reply:
Oh, great to know!